pkg/controller/blobstorage/blobstorage_controller.go

```diff
@@ -107,10 +107,26 @@ func (r *ReconcileBlobStorage) Reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, err
 	}
 
+	// Check the CR for existing Strategy
+	strategyToUse := stratMap.BlobStorage
+	if instance.Status.Strategy != "" {
+		strategyToUse = instance.Status.Strategy
+		if strategyToUse != stratMap.BlobStorage {
+			r.logger.Infof("Strategy and Provider already set, changing of cloud-resource-config config maps not allowed in existing installation. The existing Strategy is '%s' , cloud-resource-config is now set to '%s'. Operator will continue to use existing Strategy", strategyToUse, stratMap.BlobStorage)
```

Comment:
I think there's been an attempt to keep new log messages as lower case

Reply:
:+1: